### PR TITLE
[enhance] cleanup and consistency

### PR DESCRIPTION
**Pull request by julianduque**
Be consistent across the project (use commonjs instead of mixing es6 modules), and a minor typo fixed

### CODE REVIEW

File: test/helpers/message.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
   debug('init', opts);
+
   let match = opts.match || '*.';
   let user = opts.user || 'user123';
 
   return {
     match,
-    user,
+    user
```
Comment (buritica) [inline test/helpers/message.js]: please leave trailing comma?

File: test/helpers/storage.js [hunk 2]
```diff
@@ -10,8 +11,8 @@ function Storage(opts) {
   let storage = {
     users: {
       get: sinon.stub().yields(),
-      save: sinon.stub().yields(),
-    },
+      save: sinon.stub().yields()
```
Comment (buritica) [inline test/helpers/storage.js]: same

**Comment by buritica** [comment]
```
🍔  [10:03:59] buritica:leonel git:(cleanup) $ jscs lib test index.js
requireTrailingComma: Missing comma before closing curly brace at test/helpers/message.js :
    13 |  return {
    14 |    match,
    15 |    user
--------------^
    16 |  };
    17 |}

requireTrailingComma: Missing comma before closing curly brace at test/helpers/storage.js :
    12 |    users: {
    13 |      get: sinon.stub().yields(),
    14 |      save: sinon.stub().yields()
----------------------------------------^
    15 |    }
    16 |  };

requireTrailingComma: Missing comma before closing curly brace at test/helpers/storage.js :
    13 |      get: sinon.stub().yields(),
    14 |      save: sinon.stub().yields()
    15 |    }
------------^
    16 |  };
    17 |
```

**Comment by buritica** [comment]
you can also run `jscs lib test index.js --fix` to take care of any formatting issues so we keep consistency, i'll be adding checks to CI soon

**Comment by julianduque** [comment]
👌 <3

**Pull request merged**